rust-toolchain.toml

```diff
@@ -0,0 +1,14 @@
+[toolchain]
+channel = "nightly-2023-06-08"
```

Comment:
Makes `ci/env` redundant. 

However you need to manually parse this new file for all CI workflows where `ci/env` is used b/c the GitHub actions don't support it, see 
https://github.com/gluwa/creditcoin3/commit/c7f17ecb23ca72634e5b328b7eafc55d7ee71f41

Reply:
Oh I didn't know that. In that case we can actually close this PR